modules/hitl/src/views/full/message/index.jsx

```diff
@@ -74,12 +74,22 @@ export default class Message extends React.Component {
     )
   }
 
+  renderEvent() {
+    const date = moment(this.props.content.ts).format('DD MMM YYYY [at] LT')
+    return (
+      <div className={style.message + ' ' + style.event}>
+        <p>User visit : {date}</p>
+      </div>
+    )
+  }
+
   render() {
     const renderedTypes = ['text', 'message', 'image', 'video', 'audio']
 
     if (!_.includes(renderedTypes, this.props.content.type)) {
-      return null
+      return this.renderEvent()
```

Comment:
It would be better to check if `event.type` is `visit` and then update the user visit div.

Reply:
When you say it "updates the user visit div", this way there will be only 1 line of this event in the conversation and this line up at the top of conversation every time the user visit ?